pkg/service/keystore/storage.go

```diff
@@ -55,6 +55,7 @@ type ServiceKey struct {
 
 const (
 	namespace             = "keystore"
+	serviceNamespace      = "keystore:service-internal"
```

Comment:
I thought we were using `-` instead of `:` in other places, may want to make sure this is uniform